README.md

```diff
@@ -18,7 +18,10 @@ Our team was tasked with designing and creating a functioning e-commerce web app
 ### Daniel Ghaly [LinkedIn] [GitHub]
 
 ### Related Products
-### Kavita Poduri
+### Kavita Poduri | [LinkedIn](https://www.linkedin.com/in/kvpoduri/) | [GitHub](https://github.com/yrgirlkv)
+The related products widget consists of two parts: a related-products carousel and a "Your Outfit" section. The carousel renders dynamically based on the current product ID, and each card sets the current product ID to the depicted product (which accordingly rerenders the whole page.) Each card also has a button that compared the features of that product to the features of the main product displayed in the "Overview" section.
```

Comment:
compares*
(last sentence)
Keep in present tense just as your above verbs are in present tense